project/BinaryIncompatibilities.scala

```diff
@@ -6,6 +6,40 @@ object BinaryIncompatibilities {
   )
 
   val Tools = Seq(
+      ProblemFilters.exclude[MissingMethodProblem](
+        "org.scalajs.core.tools.linker.backend.emitter.ScalaJSClassEmitter.this"),
```

Comment:
Okay but it means we don't care as long as the binary incompatible methods are internal to scalajs? Only public method or runtime lib would be troublesome?